localization/languages/fr.json

```diff
@@ -66,23 +66,23 @@
         "newTabAction": "Ajouter un onglet", //this is a verb, used to label a button that adds a tab to the tabstrip when clicked
         /* task overlay */
         "viewTasks": "Voir les tâches",
-        "newTask": "Nouvelle tâche", //"new" is a verb - it is used for a button at the bottom of the task overlay
+        "newTask": "Ouvrir une nouvelle tâche", //"new" is a verb - it is used for a button at the bottom of the task overlay
         "defaultTaskName": "Tâche %n", //this is the name used for newly-created tasks; %n is replaced with a number ("task 1", "task 2", etc)
         "taskDeleteWarning": {
-            "unsafeHTML": null //missing translation
+            "unsafeHTML": "Tâche supprimée. <a>Annuler ?</a>"
         },
-        "returnToTask": null, //missing translation
-        "taskDescriptionTwo": null, //missing translation
-        "taskDescriptionThree": null, //missing translation
+        "returnToTask": "Revenir à la tâche précédente",
+        "taskDescriptionTwo": "%t et %t",
+        "taskDescriptionThree": "%t, %t, et %n",
         /* find in page toolbar */
         "searchInPage": "Rechercher dans la page", //this is used as the placeholder text for the textbox in the find in page toolbar
         "findMatchesSingular": "%i de %t match", //this and the next label are used to indicate which match is currently highlighted
         "findMatchesPlural": "%i de %t matches",
         /* Focus mode */
         "isFocusMode": "Vous êtes en mode Focus.",
         "closeDialog": "D'accord", //used as a label for the button that closes the dialog
-        "focusModeExplanation1": null, //missing translation "In focus mode, you can't create new tabs or switch tasks."
-        "focusModeExplanation2": null, //missing translation "You can leave focus mode by unchecking \"focus mode\" in the view menu."
+        "focusModeExplanation1": "En mode focus vous ne pouvez créer de nouvel onglet ni changer de tâche", //missing translation "In focus mode, you can't create new tabs or switch tasks."
```

Comment:
Can you remove the "missing translation" comments?

Reply:
Done, if I didn't miss some